integrations/slack/events/app_home_opened.go

```diff
@@ -0,0 +1,38 @@
+package events
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+type AppHomeOpen struct {
+	Type    string `json:"type,omitempty"`
+	User    string `json:"user,omitempty"`
+	Channel string `json:"channel,omitempty"`
+	Tab     string `json:"tab,omitempty"`
+	EventTS string `json:"event_ts,omitempty"`
+}
+
+type appHomeOpenContainer struct {
+	Event *AppHomeOpen `json:"event"`
+}
+
+func AppHomeOpenHandler(l *zap.Logger, w http.ResponseWriter, body []byte, cb *Callback) any {
+	// Ignore self-triggered events.
+	for _, a := range cb.Authorizations {
+		if a.UserID == cb.Event.User {
+			l.Debug("Ignoring self-triggered event")
+			return nil
+		}
+	}
```

Comment:
Unlike other events can can be triggered by live users as well as bots, this event is **always** triggered by a live user (when the user goes to the bot's "home" in Slack). Therefore this check is unnecessary in this specific event handler.